ManageIQ/manageiq
# Cinder backup actions UI

![screenshot from 2016-10-19 13 34 59](https://cloud.githubusercontent.com/assets/1280178/19530215/eecdcb44-9600-11e6-9e60-742762c5793a.png)
**Steps for Testing/QA [Optional]**

    Register an OpenStack cloud provider with Cinder backups enabled
    Go to Storage > Volumes
    Click on an existing volume
    In the "configuration" menu options to create a backup and restore from a backup should be available
